src/components/Dropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { useDebounce } from 'use-debounce';
```

Comment:
I think creating debounce hook yourself would be pretty useful, instead of installing a whole dependency